### PR TITLE
#68 add animation to popover

### PR DESCRIPTION
**Pull request by sergeymakoveev**

### CODE REVIEW

File: packages/react-kit/src/components/Popover/Popover.tsx [hunk 1]
```diff
@@ -175,27 +179,37 @@ class RawPopover extends React.Component<TFullPopoverProps, TPopoverState> {
...
-							finalAlign &&
-							hasArrow && (
-								<div
-									className={theme.arrow}
-									style={getArrowStyle(finalPlacement, finalAlign, arrowOffset)}
-								/>
-							)}
-						{this.props.children}
+			<CSSTransition
+				appear
```
Comment (morozovamv) [inline packages/react-kit/src/components/Popover/Popover.tsx]: define flags explicitly, please

Comment (sergeymakoveev) [reply packages/react-kit/src/components/Popover/Popover.tsx]: fixed in c0bda8a

File: packages/react-kit/src/components/Popover/Popover.transitions.styl [hunk 2]
```diff
@@ -0,0 +1,30 @@
+$duration = 500;
+
+.init {
+    max-height: 0;
+    transition: max-height ($duration)ms linear;
+}
+// .appear,
+// .enter {
+// }
```
Comment (sutarmin) [inline packages/react-kit/src/components/Popover/Popover.transitions.styl]: please, remove redundant comments

Comment (sergeymakoveev) [reply packages/react-kit/src/components/Popover/Popover.transitions.styl]: fixed in 83b578c

**Comment by mankdev** [comment]
close for now while it WIP

**Pull request closed without merging**